packages/incubator/Readme.md

```diff
@@ -64,30 +71,31 @@ let mut storage = MockStorage::new();
 let mut heap_store = MaxHeapStoreMut::attach_or_create(&mut storage)?;
 
 heap_store.insert(&Tx{
-    address: HumanAddr("address1".to_string()),
+    address: Addr::unchecked("address1"),
```

Comment:
I think these can be `Addr::from("...")` with MockApi, no? Just to set a better example of using it

Reply:
Addr doesn't have `::from<String>`, only HumanAddr had that in cosmwasm v0.10, now they want to make it explicit that the address was created from a string without validation that it is a correct bech32 address